.github/workflows/style_and_docs.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
         python-version: 3.9
     - name: Install Python packages
       run: |
-        pip install --upgrade pip six setuptools flake8 mypy black
+        pip install --upgrade pip six setuptools flake8 mypy>=0.800 black
```

Comment:
@trws am I correct in thinking that 0.800+ is required for our tests? I don't see any problems without this, just wanted to document the minimum version somewhere.

Reply:
You are, yes.  Likely that will increase to 801 assuming my other PR gets merged when that releases, but right now 800 is it. Good plan.